pkg/preflight/collect.go

```diff
@@ -131,6 +132,10 @@ func Collect(opts CollectOpts, p *troubleshootv1beta2.Preflight) (CollectResult,
 	collectSpecs = collect.EnsureCollectorInList(collectSpecs, troubleshootv1beta2.Collect{ClusterResources: &troubleshootv1beta2.ClusterResources{}})
 	collectSpecs = collect.EnsureClusterResourcesFirst(collectSpecs)
 
+	opts.KubernetesRestConfig.QPS = 100
+	opts.KubernetesRestConfig.Burst = 100
+	opts.KubernetesRestConfig.UserAgent = fmt.Sprintf("ReplicatedTroubleshoot/%s", version.Version())
```

Comment:
I guess my only suggestion might be to define this string as a constant and consume it from a variable, but I'm not sure where that should live

Reply:
oh probably `constants.go` 😂

Reply:
Thank you @adamancini ! Yes, I have added constants.go and changed to
```
opts.KubernetesRestConfig.QPS = constants.DEFAULT_CLIENT_QPS
opts.KubernetesRestConfig.Burst = constants.DEFAULT_CLIENT_BURST
opts.KubernetesRestConfig.UserAgent = fmt.Sprintf("%s/%s", constants.DEFAULT_CLIENT_USER_AGENT, version.Version())
```